app/controllers/ncr/work_orders_controller.rb

```diff
@@ -4,12 +4,11 @@ class WorkOrdersController < UseCaseController
     MAX_UPLOADS_ON_NEW = 10
 
     def new
-      @approver_email = self.suggested_approver_email
+      @model_instance.approving_official_email = self.suggested_approver_email
```

Comment:
> I tend to use `self` for class methods only - thoughts on just using `suggested_approver_email` here?

I use `self` for calling instance methods to distinguish them from variable references, but I acknowledge that isn't a common pattern in Ruby. The reasoning, if you're curious is that I've been burned by bugs like

```ruby
something = 'foo'
# sets a local variable, where the intended behavior was actually
self.something = 'foo'
```

Using `self` just avoids that ambiguity. Starting to lean towards only using `self` for assignment methods, though. Will remove.

> is this controller being used for more than one type of object?

This inherits from the UseCaseController, where `@model_instance` is set.

> I think something like `@ncr_work_order` would be a clearer var name than `@model_instance`

I hear you, but mind if we take care of that separately? If nothing else, because there are several open PRs that change WorkOrdersController, and :crying_cat_face: :weary: :-1: more merge conflicts.

/cc https://www.pivotaltracker.com/story/show/101710542